src/test/java/org/apache/maven/plugins/gpg/GpgVersionTest.java

```diff
@@ -19,19 +19,30 @@
  * under the License.
  */
 
+import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 
-import org.apache.maven.plugins.gpg.GpgVersion;
 import org.junit.Test;
 
+/**
+ * Tests for {@link GpgVersion}.
+ */
 public class GpgVersionTest
 {
     @Test
     public void test()
     {
-        assertTrue( GpgVersion.parse( "gpg (GnuPG) 2.2.1" ).isAtLeast( GpgVersion.parse( "gpg (GnuPG) 2.2.1" ) ) );
-        assertTrue( GpgVersion.parse( "gpg (GnuPG) 2.2.1" ).isAtLeast( GpgVersion.parse( "2.1" ) ) );
-        assertTrue( GpgVersion.parse( "gpg (GnuPG/MacGPG2) 2.2.10" ).isAtLeast( GpgVersion.parse( "2.2.10" ) ) );
+        assertEquals( GpgVersion.parse( "gpg (GnuPG) 2.2.1" )
```

Comment:
This is probably three separate test methods, not one.

Reply:
I don't know what three methods you mean.
There were 3 assertions and now there are 4.
In this PR i added one assertion statement. And i made the above existing assertions more assertive, this means more sensitive for a change (turned some isAtLeast to compareTo = 0).